src/g2c_interface.F90

```diff
@@ -0,0 +1,36 @@
+! Contains a module with interfaces to the C functions in the g2c
+! library.
+!
+! Note: this file is excluded from the doxygen build (see Doxyfile.in)
+! because these functions are already documented in the g2c library.
```

Comment:
Does/should the doxygen docs have some indication of which functions have Fortran interfaces?

Reply:
No users access everything through the fortran API and never call these functions directly.